### PR TITLE
feat(docs): scons plugin reference

### PR DESCRIPTION
**Pull request by sergiusens**
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

File: docs/common/craft-parts/reference/plugins/scons_plugin.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. _craft_parts_scons_plugin:
+
+Scons plugin
+============
+
+The Scons plugin builds projects using Scons_.
+
+After a successful build, this plugin will installs the generated
```
Comment (lengau) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:8]:
```suggestion
After a successful build, this plugin will install the generated
```

File: docs/common/craft-parts/reference/plugins/scons_plugin.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.. _craft_parts_scons_plugin:
+
+Scons plugin
```
Comment (tigarmo) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:3]: I believe it's `SCons`

File: docs/common/craft-parts/reference/plugins/scons_plugin.rst (hunk 1 above)
Comment (tigarmo) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:8]:
```suggestion
After a successful build, this plugin will install the generated
```

File: docs/common/craft-parts/reference/plugins/scons_plugin.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Environment variables
+---------------------
+
+This plugin sets ``DESTDIR`` to ``$CRAFT_PART_INSTALL``.
+
+Dependencies
+------------
+
+The Scons plugin needs the ``scons`` executable work.
```
Comment (tigarmo) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:33]:
```suggestion
The Scons plugin needs the ``scons`` executable to work.
```

Comment (tigarmo) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:33]: should also mention that the plugin itself does NOT provision scons

Comment (lengau) [inline docs/common/craft-parts/reference/plugins/scons_plugin.rst:33]:
```suggestion
The Scons plugin needs the ``scons`` executable to work.
```

Should we mention that the plugin doesn't install it by default?

**Review by tigarmo — CHANGES_REQUESTED** [review]

File: docs/common/craft-parts/reference/plugins/scons_plugin.rst (hunk 3 above)
Comment (sergiusens) [reply docs/common/craft-parts/reference/plugins/scons_plugin.rst:33]: yeah, I recalled this when I was picking up my kids. I was wondering why I had not submitted it by then. thanks!

**Review by lengau — COMMENTED** [review]
Thanks!

**Review by tigarmo — APPROVED** [review]

**Pull request merged**